src/extension/src/EnvManager.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2020 Microsoft Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Requires Python 2.7+
+
+
+class EnvManager(object):
```

Comment:
Try to make this more specific as naming influences whether this becomes a dumping ground for things over time (bad), vs. forcing creation of specific classes as needed (good). EnvHealthManager for example.. or EnvIssueMitigator (not suitable because it doesn't allude to detection) or any others..

Reply:
Used EnvHealthManager. I thought about EnvIssueMitigator while writing this code but it didn't completely justify the actions, so had stuck to EnvManager. EnvHealthManager sounds good